tests/drivers/build_all/ethernet/spi_devices.overlay

```diff
@@ -110,13 +119,15 @@
 
 					ethernet-phy@1 {
 						reg = <0x1>;
-						compatible = "adi,adin2111-phy";
+						compatible = "realtek,rtl8211f";
```

Comment:
this doesn't make any sense to test the build like this for adin2111, the adin2111 is a chip with an integrated mac and a couple phys, instead of changing this, you should make a new node and test case for the other phys if you want to test building them

Reply:
Thank you for your review.
As you say, there is no need to change existing tests.
I created a "vnd,ethernet" definition for testing and put the build tests for the phy driver there.

Reply:
Currently, the following dummy devices exist.

> drivers/pwm/pwm_test.c
> drivers/spi/spi_test.c
> drivers/i2c/i2c_test.c
> drivers/gpio/gpio_test.c
> drivers/adc/adc_test.c
> drivers/serial/serial_test.c
> drivers/w1/w1_test.c
> drivers/dac/dac_test.c
> drivers/i3c/i3c_test.c

These are used outside of tests/drivers/build_all, 
(e.g. from tests/lib/devicetree/api/) So I think it would be better to keep the current file location.